firebase-vscode/src/workflow.ts

```diff
@@ -0,0 +1,429 @@
+import * as path from "path";
+import * as fs from "fs";
+import * as vscode from "vscode";
+import { ExtensionContext, workspace } from "vscode";
+import { FirebaseProjectMetadata } from "../../src/types/project";
+import { writeFirebaseRCFile } from "./utils";
+import { ExtensionBrokerImpl } from "./extension-broker";
+import {
+  deployToHosting,
+  getAccounts,
+  listProjects,
+  login,
+  logoutUser,
+  initHosting,
+  getChannels,
+} from "./cli";
+import { User } from "../../src/types/auth";
+import { FirebaseRC } from "../../src/firebaserc";
+import { FirebaseConfig } from "../../src/firebaseConfig";
+import { currentOptions, updateOptions } from "./options";
+import { ServiceAccountUser } from "./types";
+import { selectProjectInMonospace } from "../../src/monospace";
+import { setupLoggers, tryStringify } from "../../src/utils";
+import { pluginLogger } from "./logger-wrapper";
+import { logger } from '../../src/logger';
+import { transports, format } from "winston";
```

Comment:
Same suggestion as before, convention is to split external and internal imports. Totally ok if you want to wait for me to add the linter rule